scripts/deploy_protocol.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import asyncio
+from starknet_py.net.account.account_client import (AccountClient)
+from starknet_py.net.signer.stark_curve_signer import KeyPair
+from starknet_py.contract import Contract
+from starknet_py.net.models import StarknetChainId
+from starkware.crypto.signature.signature import private_to_stark_key
+from starknet_py.net.gateway_client import GatewayClient
+from utils import deployContract
+from pathlib import Path
+
+#Configure Admin AccountClient
+private_key = int(sys.argv[1])
+account_address = int(sys.argv[2],16)
+public_key = private_to_stark_key(private_key)
+signer_key_pair = KeyPair(private_key,public_key)
+client = AccountClient(address=account_address, client=GatewayClient(net="testnet"), key_pair=signer_key_pair, chain=StarknetChainId.TESTNET, supported_tx_version=1)
+kakarot_abi = [
```

Comment:
can this be imported from kakarot_abi.json?

Reply:
Aggreed